scaffolding-chef-inspec/tests/user-linux-api/habitat/plan.sh

```diff
@@ -0,0 +1,5 @@
+pkg_name=user-linux-default
+pkg_version=0.1.0
+pkg_origin=chef
+pkg_scaffolding="ci/scaffolding-chef-inspec"
```

Comment:
I don't understand how the "ci" origin works for the `pkg_scaffolding="ci/scaffolding-chef-inspec"`

Reply:
I kind of see what is happening here. The test builds "scaffolding-chef-inspec" with the "ci" origin and then the test is run with that build.

Reply:
Yep this is cause we need to verify the scaffolding using a user plan but I don't think we want to build something that would go into unstable on every PR.